data/bootstrap_hwdefinition.yaml

```diff
@@ -0,0 +1,65 @@
+---
+#############################################################################################################################
```

Comment:
We've been standardizing on Apache 2.0 licenses, such as:

https://github.com/att-comdev/openstack-helm/blob/master/ceph/Chart.yaml